### DISCUSSION TITLE
Add a setting to disable forking and clones/downloading ZIPs of special repos

### DISCUSSION THREAD

**Discussion by GhostOf0days** [community/community#61679] [Other Feature Feedback, Questions, & Ideas]
### Select Topic Area

Product Feedback

### Body

Special repos (public repos that are username/username and used as an about me) often contain private information that the user may want to remove from their repo later. For this reason, you should be able to disable forking, clones, and downloading ZIPs of special repos as they contain private information the user may want to remove later. Also, an option to disable viewing the raw code of the ReadMe for people who aren't the owners. However, these settings should be made as options that can be disabled as not everyone uses the special repos as about me's and sometimes just put stuff like dynamic games in them.

**Comment by dannz510** [comment]


**Reply by GhostOf0days** [reply]
Github does not allow you to manage the forking policy of special repos. Special repos are not organization repos. Special repos are those public repos that are public and are the username of someone. For example, example/example, where the first is the repo owner and the second is the repo name. They usually serve as about me's. You can not manage this.

**Comment by uncenter** [comment]
First of all, don't put any private information that you want to remove in a public repository, especially one that is displayed on your profile! It's a public Git repository, same as any other one - no reason to prevent any of that from GitHub's end. Second, you can always delete the repository and make a new one to remove the history...

**Reply by GhostOf0days** [reply]
? Special repos are literally meant to display about me information. Look up what a special repo is.

**Reply by uncenter** [reply]
It's quite literally a normal repository that GitHub displays the README of on your profile... I looked up like you said and found *nothing*.

**Reply by GhostOf0days** [reply]
No it isn't. Try to create a public repo that is the same name of your profile. https://docs.github.com/en/account-and-profile/setting-up-and-managing-your-github-profile/customizing-your-profile/managing-your-profile-readme A lot of people who have used GitHub for longer have these. They are used as about me's. They should not be forkable.

**Reply by uncenter** [reply]
... I already have that set up. I use it, and I know how it works. The article you linked me to literally says:

<img width="736" alt="image" src="https://github.com/community/community/assets/47499684/69423053-0386-4381-a792-6f9dbf397cce">

It's no different than any other public repository.

**Reply by GhostOf0days** [reply]
Yes. However, my point is those repos are often used as about me's. They often contain information that should not be forkable in case the user decides to make it private at a later point, effectively removing their 'about me' repo.

**Reply by uncenter** [reply]
If you have such a large online presence that people are constantly forking your about me repository, then you should be very careful about what you put in it in the first place! I'm also confused why this is such an important concern for you seeing you just have music and your name in your README and 0 forks.

**Reply by GhostOf0days** [reply]
I'm going to end the convo here as I don't want to continue it. However, someone did fork it. It's more so a privacy matter if you later decide to remove information about yourself from your public Github account.

**Reply by uncenter** [reply]
I'm all for privacy. I think being able to remove private information like that is important but unfortunately you'll probably just have to be more careful in the future and try to reach out to anyone who has forked it and request them to delete it.
